tmail-backend/blob/blob-guice/src/main/java/com/linagora/tmail/blob/guice/BlobStoreConfiguration.java

```diff
@@ -41,7 +19,8 @@
 public record BlobStoreConfiguration(boolean cacheEnabled,
                                      StorageStrategy storageStrategy,
                                      Optional<CryptoConfig> cryptoConfig,
-                                     boolean singleSaveEnabled) {
+                                     boolean singleSaveEnabled,
+                                     Optional<S3BlobStoreConfiguration> maybeSecondaryS3BlobStoreConfiguration) {
```

Comment:
I feel the put `S3BlobStoreConfiguration` directly here (in BlobStoreConfiguration) is should not. 
`BlobStoreConfiguration` is more generic to blobStore, not stick to special implement

@Team 
WDYT?